### ISSUE TITLE
Selenium grid support

### ISSUE THREAD

**Issue by carlosnizolli** [MarketSquare/robotframework-browser#2194]
**Is your feature request related to a problem? Please describe.**
Run parallel tests in remote machine. Remote browser 

**Describe the solution you'd like**
Run with selenium grid
https://playwright.dev/docs/selenium-grid

**Describe alternatives you've considered**
Remote browser without websocket

**Additional context**

**Comment by aaltat** [comment]
Yes, that should work currently out of the box. What is the missing feature?

**Comment by carlosnizolli** [comment]
> Yes, that should work currently out of the box. What is the missing feature?


How does it work? I didn't find kw for this, just the "Connect To Browser" that uses wsEndpoint

**Comment by aaltat** [comment]
As Playwright documentation explains, one just declares the required environment variables, before launching test execution and all should work out of the box.

**Comment by carlosnizolli** [comment]
> As Playwright documentation explains, one just declares the required environment variables, before launching test execution and all should work out of the box.

This solution worked. Thanks

**Comment by Aino1980** [comment]
> > As Playwright documentation explains, one just declares the required environment variables, before launching test execution and all should work out of the box.
> 
> This solution worked. Thanks

how solution worked. Thanks
